src/components/Widgets/DateControl.js

```diff
@@ -0,0 +1,21 @@
+import React, { PropTypes } from 'react';
+import DateTime from 'react-datetime';
+
+export default class DateControl extends React.Component {
+  handleChange = (datetime) => {
+    this.props.onChange(datetime);
+  };
+
+  render() {
+    return (<DateTime
+      timeFormat={false}
+      value={this.props.value || new Date()}
+      onChange={this.handleChange}
+    />);
+  }
+}
+
+DateControl.propTypes = {
+  onChange: PropTypes.func.isRequired,
+  value: PropTypes.object,
```

Comment:
Our linter treats `PropTypes.object` as an error, but in this case I'm not sure there's a better option.

The DateControl (and preview) should be able to handle either a string, a Date or a moment.js object, since this will vary depending on the formatter (JSON and CSV for example, doesn't have any date concept and will deserialize date fields to a string).